src/main/kotlin/lotto/LottoUser.kt

```diff
@@ -0,0 +1,28 @@
+package lotto
+
+import kotlin.math.round
+
+class LottoUser {
+    val lottos: List<Lotto>
+        get() = _lottos
+    private val _lottos = arrayListOf<Lotto>()
+
+    fun purchaseLotto(enableLottoPurchaseCount: Int) {
+        _lottos.clear()
+        _lottos.addAll(LottoSeller.sellLotto(enableLottoPurchaseCount))
+    }
+
+    fun calculateWinningMoney(winningNumber: List<Int>): Int {
+        var winningMoney = 0
+        _lottos.forEach { lotto ->
+            val sameCount = lotto.getContainLottoNumberSameCount(winningNumber)
+            winningMoney += lotto.exchangeWinningMoney(sameCount)
+        }
+
+        return winningMoney
+    }
```

Comment:
```suggestion
    fun calculateWinningMoney(winningNumber: List<Int>): Int {
        return _lottos.sumOf { lotto ->
            val sameCount = lotto.getContainLottoNumberSameCount(winningNumber)
            lotto.exchangeWinningMoney(sameCount)
        }
    }
```
kotlin collections API의 [sumOf](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/sum-of.html)을 활용하시면 variable을 선언하지 않으셔도 됩니다 🙂